docs/env.md

```diff
@@ -0,0 +1,10 @@
+## Environment variables
+
+### Change server address
+    BRCEP_ADDRESS=:8000
```

Comment:
why tabs at the start of the line?

Reply:
the tabs are added to make the line look as a code block

Reply:
This is flat config file, not need markdown format.